### ISSUE TITLE
Error when trying to load the kext

### ISSUE THREAD

**Issue by cstefann** [AppleIntelWifi/device-logs#33]
I've downloaded the kext from the website (Relase version) and when i try to load it i get this error.
macOS version: macOS Catalina 10.15.4
laptop: Lenovo 530s-15ikb
![Screenshot 2020-04-18 at 12 44 40](https://user-images.githubusercontent.com/33597024/79634471-3940ed80-8173-11ea-8a52-e1113a85d512.png)
Here is the log: 
[log.txt](https://github.com/AppleIntelWifi/device-logs/files/4496402/log.txt)

**Comment by rv32ima** [comment]
Fixed, please try again

**Comment by cstefann** [comment]
Now it works, but when i search for networks, the network list is empty. Kext loads perfectly fine now. I opened yesterday an new issue topic. I'll be back with some logs when i have some free time. Thank you for all the work!

**Comment by rv32ima** [comment]
Yeah.. the network list is something I have to try to figure out. There's no publicly available documentation on any of this, so I have to do a large amount of trial and error.

**Comment by cstefann** [comment]
If i can help you with something, i'm here.
I uploaded right now some logs on the other topic, hope this will help with the development.
